.github/workflows/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- Kindly remove the .gitignore file from this directory you only need to add it once and that is in the root directory 👨‍🌾.